modules/attribution-document/attribution-document-core/src/main/java/org/eclipse/sw360/antenna/attribution/document/core/Templates.java

```diff
@@ -99,6 +99,12 @@ public PDFont loadBoldItalicFont(PDDocument doc) {
                 return IOUtils.toByteArray(fontData.get());
             } catch (IOException e) {
                 throw new ExecutionException("Unable to load font.", e);
+            } finally {
+                try {
+                    fontData.get().close();
```

Comment:
Could we also solve this by using *try-with-resources* in line 98 on `fontData.get()`.